ui/src/main/java/org/eclipse/starter/ui/Project.java

```diff
@@ -330,7 +347,10 @@ public void generate() {
 	}
 
 	private String getCacheKey() {
-		return jakartaVersion + ":" + profile + ":" + javaVersion + ":" + docker + ":" + runtime;
+		// if groupid or artifactid was changed, don't use cache
```

Comment:
What's the need to add these to the cache key? The longer the key, the more complex the hash calculation.